.gitignore

```diff
@@ -2,3 +2,5 @@ node_modules
 .idea
 persist
 config.json
+package-lock.json
```

Comment:
[package-lock.json should be committed](https://github.com/npm/npm/blob/v5.0.0/doc/files/package-lock.json.md)